ui/src/components/connect/ConnectPage.js

```diff
@@ -5,50 +5,52 @@ import {
   Image,
   VStack,
   useMediaQuery,
-} from "@chakra-ui/react";
-import ExperienceHmcc from "./ExperienceHmcc";
-import Faq from "./Faq";
-import OurMinistries from "./OurMinistries";
-import LifeGroups from "./LifeGroups";
+  Flex,
+} from '@chakra-ui/react';
+import ExperienceHmcc from './ExperienceHmcc';
+import Faq from './Faq';
+import OurMinistries from './OurMinistries';
+import LifeGroups from './LifeGroups';
 
 const ConnectPage = (props) => {
-  const [isLargerThan768] = useMediaQuery("(min-width: 768px)");
+  const [isLargerThan768] = useMediaQuery('(min-width: 768px)');
   return (
     <Box background="linear-gradient(151.15deg, rgba(255, 244, 201, 0.3) 11.18%, rgba(255, 255, 255, 0.3) 42.46%, rgba(202, 220, 255, 0.3) 76.7%), linear-gradient(194.34deg, #FFE6E6 1.83%, #FFFFFF 51.22%, #D6FFEA 99.59%)">
       <Container maxW="container.lg" py={10}>
         <VStack spacing={[4, 12]} align="stretch">
-          <Box>
-            <Heading
-              as="h1"
-              fontSize={["2.25em", "3.5em", "6em"]}
-              fontWeight={800}
-              textAlign="left"
-            >
-              Welcome!
-            </Heading>
-            <Heading
-              as="h2"
-              fontSize={["1.5m", "2.5em", "4em"]}
-              fontWeight={800}
-              textAlign="left"
-            >
-              We're so glad you're here :)
-            </Heading>
-          </Box>
-          {isLargerThan768 ? (
+          <Flex flexDirection={isLargerThan768 ? 'row' : 'column'} mt={7}>
```

Comment:
![image](https://user-images.githubusercontent.com/33679483/183928905-670e88a2-360e-4ee6-86c6-cd3d4df956de.png)

- FAQs dont match the Figma?
- flip tile not mobile responsive

Reply:
![image](https://user-images.githubusercontent.com/33679483/183929779-fcc6b019-e47f-4512-999e-da85e2b89532.png)

this looks off. it should have some space between "MARRIED COUPLES" and the description